pandas/core/arrays/sparse.py

```diff
@@ -397,8 +399,12 @@ def _get_fill(arr):
         return np.asarray(arr.fill_value)
 
 
-def _sparse_array_op(left, right, op, name):
-    # type: (SparseArray, SparseArray, Callable, str) -> Any
+def _sparse_array_op(
+        left: 'SparseArray',
```

Comment:
ABCSparseArray

Reply:
I think we're good to go @WillAyd @jreback